requirements.txt

```diff
@@ -16,11 +16,12 @@ pygments==2.2.0
 pytest-runner>=3.0.0,<3.1.0
 pytest>=3.2.0,<3.3.0
 PyYaml==3.13
-readme-renderer==17.2
-setuptools==37.0.0
+readme-renderer>=24.0
```

Comment:
We should maybe stick with `==` (or add an upper bound) here to avoid pulling in an incompatible dependency in the future.